examples/lvgldemo/tp_cal.h

```diff
@@ -62,7 +47,7 @@ extern "C"
 void tp_cal_create(void);
 
 #ifdef __cplusplus
-} /* extern "C" */
-#endif
+}
+#endif  /* extern "C" */
```

Comment:
```suggestion
#endif
```
as there is no such comment in other files